lang/ghc-ppc-bootstrap/Portfile

```diff
@@ -51,8 +52,8 @@ extract {
     system -W ${worksrcpath}/files/ghc.pkg "cat Payload | gunzip -dc | cpio -i"
     
 if {![variant_isset framework]} {
-    system -W ${worksrcpath}/files/ghc.pkg "mv GHC.framework/Versions/7.0.4-powerpc/usr ${worksrcpath}/${name}"
-    system -W ${worksrcpath}/files/ghc.pkg "cp -R GHC.framework/Versions/7.0.4-powerpc/Tools ${worksrcpath}/${name}/share"
+    system -W ${worksrcpath}/files/ghc.pkg "mv GHC.framework/Versions/${version}-powerpc/usr ${worksrcpath}/${name}"
+    system -W ${worksrcpath}/files/ghc.pkg "cp -R GHC.framework/Versions/${version}-powerpc/Tools ${worksrcpath}/${name}/share"
```

Comment:
Not related to the changes you're making here of course, but MacPorts does have `move` and `copy` commands that should usually be used instead of shell commands.

Reply:
This code was largely based upon existing previously in Macports. Can be simplified, but then it makes sense to do for `ghc-bootstrap` as well, and that is quite a huge port to rebuild (and also rather fragile). Unless it is a matter of principle for someone, I rather leave it as it is, at least until some major refactoring is done for both ports.

Reply:
@pmetzger I agree.

Reply:
Okay @barracuda156 I'm going to commit. Please go through and fix the tcl in the portfiles to use the move and copy commands and do a distinct pull request for it.